common/main/be_TARASK.xml

```diff
@@ -11,7 +11,7 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 	<identity>
 		<version number="$Revision$"/>
 		<language type="be"/>
-		<variant type="tarask"/>
+		<variant type="TARASK"/>
```

Comment:
should be a test for this

Reply:
Filed as https://unicode-org.atlassian.net/browse/CLDR-14978